charts/nublado2/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: jupyterhub
+  version: ">=0.1.0"
```

Comment:
We should pin this to a specific version. Otherwise, testing with a specific version of the chart is not deterministic.

Reply:
...and that's an ancient version.  Our image pins 1.2, so that seems like a good start.